src/community/model_deployments/hugging_face.py

```diff
@@ -1,11 +1,10 @@
 import logging
-from typing import Any, Dict, Generator, List
+from typing import Any, Dict, List
 
-from cohere.types import NonStreamedChatResponse, StreamedChatResponse
 from transformers import AutoModelForCausalLM, AutoTokenizer
 
+from backend.model_deployments.base import BaseDeployment
```

Comment:
Should we import this from community as it was before?

Reply:
done